lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+class Customer
+  require 'csv'
+  
+  attr_reader :id
+  attr_accessor :email, :address
+  
+  
+  
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+  
+  def self.all
+    all_customers = []
+    customer_data = CSV.open('data/customers.csv', headers:false).map(&:to_a)
+    customer_data.each do |customer|
+      parameter1 = customer[0].to_i
```

Comment:
This is another example where I'd suggest coming up with more descriptive variable names.